src/routes/landing/components/CityEntry.js

```diff
@@ -23,40 +23,49 @@ const CityListItem = styled(Link)`
     background-color: ${props => props.theme.colors.backgroundAccentColor};
   }
 `
-const MunicipalityItem = styled(Highlighter)`
+
+const AliasItem = styled(Highlighter)`
+  display: inline-block;
   font-size: 12px;
+
+  &:not(:last-child):after {
+    content: ',\\00a0';
```

Comment:
Pls add a comment what this is used for. Instead, you could also use the same approach as in the native app, i.e. add the comma and whitespace directly to the text below. I'd prefer the latter, because imo its clearer.